hazardous/_gradient_boosting_incidence.py

```diff
@@ -84,6 +84,21 @@ def draw(self):
 class GradientBoostingIncidence(BaseEstimator, ClassifierMixin):
     """Cause-specific Cumulative Incidence Function (CIF) with GBDT.
 
+    In competing risk analysis, one wants to predict each cause-specific
+    Cumulative Incidence Function that is to say:
+
+    .. math::
+        \hat{F}_k(t) \approx \mathbb{P}(T \leq t, E= k)
+
+    One can obtain the survival probability of any event by computing
+    each CIF (that is to say running the model
+    for each cause-specific CIF) and computing 1 - “sum of CIF curves” because:
+
+    .. math:
+        S(t) = \mathbb{P}(T > t) = 1 - \mathbb{P}(T \leq t)
+        1 - \sum_{k=1}^K \mathbb{P}(T \leq t, E= k)
```

Comment:
Does not impact the rendering, just for consistency:

```suggestion
        1 - \sum_{k=1}^K \mathbb{P}(T \leq t, E=k)
```